src/main/java/com/nylas/NylasClient.java

```diff
@@ -140,28 +161,44 @@ public NylasAccount account(String accessToken) {
 	 * https://square.github.io/okhttp/4.x/okhttp/okhttp3/-response-body/#the-response-body-must-be-closed</a>
 	 */
 	ResponseBody download(String authUser, HttpUrl.Builder url) throws IOException, RequestFailedException {
-		Request request = buildRequest(authUser, url, HttpMethod.GET, null);
+		Request request = buildRequest(authUser, url, HttpMethod.GET, null, AuthMethod.BASIC);
 		Response response = getHttpClient().newCall(request).execute();
 		throwAndCloseOnFailedRequest(response);
 		return response.body();
 	}
 
 	<T> T executeRequestWithAuth(String authUser, HttpUrl.Builder url, HttpMethod method, RequestBody body,
 			Type resultType) throws IOException, RequestFailedException {
-		Request request = buildRequest(authUser, url, method, body);
+		return executeRequestWithAuth(authUser, url, method, body, resultType, AuthMethod.BASIC);
+	}
+
+	<T> T executeRequestWithAuth(String authUser, HttpUrl.Builder url, HttpMethod method, RequestBody body,
+								 Type resultType, AuthMethod authMethod) throws IOException, RequestFailedException {
```

Comment:
nit: why is this line tabbed over so far...?

Reply:
My Intellij settings were out of whack since the upgrade, fixed now! ✅